wgsl/index.bs

```diff
@@ -1585,6 +1585,46 @@ Note: When no conversion is performed, the conversion rank is zero.
       <td>ConversionRank(|S|,|T|)
       <td>Inherit conversion rank from component type.
       Note: Only [=fixed-size arrays=] may have an [=type/abstract=] component type.
+  <tr algorithm="conversion rank for frexp result">
+      <td>__frexp_result_abstract
+      <td>__frexp_result
+      <td>1
+      <td>Inherit conversion rank from input type.
```

Comment:
These "Inherit conversion rank from input type" don't make sense.  I think delete them.
The conversion rank is given explicitly.